src/store/state/state-model.ts

```diff
@@ -184,7 +185,7 @@ export const stateModel: StateModelIF = {
       routingSlipNumber: '',
       bcolAccountNumber: '',
       datNumber: '',
-      folioNumber: '',
+      folioNumber: '', // *** TODO: FN4 is here
```

Comment:
I plan to combine a couple of these. When I'm done, there should remain:
- the FN from the account
- the transactional FN (ie, override)
- the staff payment FN (which is mutually exclusive with the TFN)